### ISSUE TITLE
Notification System - ability to hide scanner errors

### ISSUE THREAD

**Issue by ClumsyAdmin** [photoview/photoview#563]
**Is your feature request related to a problem? Please describe.**
Since some of my photos aren't supported by darktable, I always get scanner errors on each scan that cover my screen and such.

**Describe the solution you'd like**
I would like it if I had the ability to disable scanner errors so it doesn't spam my screen.

**Describe alternatives you've considered**
Perhaps don't attempt rescanning photos that have been scanned in the past and resulted in scanner errors.

**Additional context**
N/A

**Comment by dansity** [comment]
I agree, it is bonkers. I have 600+ errors after each scan that I should click X one-by-one. I rather just close photoview, I'm not going to click them. I can compromise and have a single popup saying "Scan finished. You have xxx number of errors. Click here to see log."

**Comment by PJ-Watson** [comment]
> **Describe alternatives you've considered** Perhaps don't attempt rescanning photos that have been scanned in the past and resulted in scanner errors.
>
One issue I can see here is that if the user fixes something that caused the scanner error, it wouldn't be picked up unless the path to the photo also changes. Not sure there's an easy way round that though, and I suspect user-fixable errors are also vanishingly rare.

**Comment by kkovaletp** [comment]
I agree with the proposal to implement an option to turn such notifications off. It should only affect the current user, be located on the settings page, and be enabled by default so a user can disable them if needed.

> have a single popup saying "Scan finished. You have xxx number of errors. Click here to see log."

I don't see a value here, as:
- this notification is not informative much
- all scanner issues are written in the server log and can be accessed at any time from the server, but there is no way to access them from Photoview UI, and I don't see a value in this feature

> don't attempt rescanning photos that have been scanned in the past and resulted in scanner errors.

This is a bad idea, as (also mentioned in previous comments) it would make those media invisible for Photoview, even it was fixed later.

BTW, we have #996 and #939 to improve the scanner job
